src/Http/ViewComposers/ConfigComposer.php

```diff
@@ -57,6 +58,7 @@ public function getConfig(): array
             'show_tax'                     => (bool) Rapidez::config('tax/display/type', 1),
             'grid_per_page'                => Rapidez::config('catalog/frontend/grid_per_page', 12),
             'grid_per_page_values'         => explode(',', Rapidez::config('catalog/frontend/grid_per_page_values', '12,24,36')),
+            'max_category_level'           => Category::withoutGlobalScopes()->selectRaw('MAX(level) as maxlevel')->first()->maxlevel,
```

Comment:
Should we cache this? It's just a simple quick query, but Redis is always faster.

Reply:
I thought this whole config composer got cached, but it looks like it does not? In that case, yeah it should be cached. But shouldn't we cache the whole thing?

Reply:
All the individual `Rapidez::config()`'s are cached bundled with one cache key. This config could be dynamic; for example the checkout GraphQL queries are just useful for checkout pages. So just caching the output with `Cache::rememberForever()` is fine with me.